core/src/main/java/com/linecorp/armeria/server/management/ThreadDumpService.java

```diff
@@ -54,7 +55,7 @@ public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) throws Exc
     }
 
     @Override
-    public ExchangeType exchangeType(RequestHeaders headers, Route route) {
+    public ExchangeType exchangeType(RequestHeaders headers, Routed<ServiceConfig> routed) {
```

Comment:
I'm not 100% sure, but can we pass `RoutingContext` instead?

Reply:
Possible if `RoutingContext.result()` is added.